dashboard/app/views/levels/editors/fields/_special_variables.html.haml

```diff
@@ -0,0 +1,6 @@
+%h1.control-legend{data: {toggle: "collapse", target: "#special_vars_area"}}
```

Comment:
should we make this section and file name more specific? Special Variables doesn't imply this file is csa-specific and might cause confusion down the line. @jmkulwik do you know if theater will have a similar checkbox configuration or if we are going to need more configuration options for neighborhood/theater? I'm wondering if this file should either be titled relating to Neighborhood or to CSA.

Reply:
I'm on board for whichever name! I remember Jessie mentioning that it could be useful to have an area for more than just this neighborhood checkbox, but agree that the name is possibly too generic. We could do something like "CSA Features"?

Reply:
looking at various field files, looks like `_csa_fields.html.haml` would fit with our current pattern. For the section name (as in what the user sees), I think CSA Options or CSA Features makes sense, but others may have a stronger opinion.

Reply:
Theater will have a section for multimedia asset upload https://docs.google.com/document/d/1m7s_DF2JM4ZRbbihj3HsuAA3n-wePegLvcwFQjEx2sA/edit#bookmark=id.q08wm749igva 

Theater will have a similar option. Looking at the design doc ^ I'm realizing this is intended to be a dropdown instead so it can switch between the three main views: console/neighborhood/theater. Sorry for the switch on this, but we'll want to adjust this to a dropdown with three options rather than a simple checkbox. Jillian recently did something similar here: https://github.com/code-dot-org/code-dot-org/pull/40284

Reply:
Okay- so keeping track of an enum instead? And does the header "CSA Features" still make sense?

Reply:
CSA Features sounds good to me! And yes, enum instead of bool.